python_bindings/apps/CMakeLists.txt

```
@@ -5,11 +5,13 @@ set(SCRIPTS
     interpolate.py
     local_laplacian.py)
 
+make_shell_path(TEST_IMAGES_DIR $<SHELL_PATH:${CMAKE_CURRENT_SOURCE_DIR}/../../apps/images>)
```

For a single directory, you don't need to use `make_shell_path`. Once we bump the minimum CMake version to 3.22, we can get rid of the helper altogether! 😄

---

Also this list of environment variables is getting long enough to split onto multiple lines.

```suggestion
set(test_env
    "PYTHONPATH=$<SHELL_PATH:$<TARGET_FILE_DIR:Halide::Python>>"
    "HL_TARGET=${Halide_TARGET}"
    "TEST_TMPDIR=$<SHELL_PATH:${CMAKE_CURRENT_BINARY_DIR}>"
    "TEST_IMAGES_DIR=$<SHELL_PATH:${CMAKE_CURRENT_SOURCE_DIR}/../../apps/images>"
)
```